src/cow-react/modules/limitOrders/updaters/ActiveRateUpdater/index.ts

```diff
@@ -43,17 +43,12 @@ export function ActiveRateUpdater() {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [activeRate])
 
-  useEffect(() => {
+  useLayoutEffect(() => {
     // Clear active rate on network change
     if (prevChainId && prevChainId !== chainId) {
       updateLimitRateState({ activeRate: null })
     }
-
-    // Clear active rate when its loading (currency changed)
-    if (isLoading) {
```

Comment:
@nenadV91 it looks unnecessary and created blinking on the form, so, I just deleted it :)
Can it be a problem?